core/src/main/java/org/akaza/openclinica/dao/core/CoreResources.java

```diff
@@ -363,7 +363,7 @@ private Properties setDataInfoProperties() {
 
         String supportURL = DATAINFO.getProperty("supportURL");
         if (supportURL == null || supportURL.isEmpty())
-            supportURL = "http://www.openclinica.org/OpenClinica/3.1/support/";
+            supportURL = "https://www.openclinica.com/support/";

```

Comment:
@kkrumlian but it's already that? (in datainfo.properties)
Or do you mean that I should remove the trailing slash?